ibis/backends/pandas/execution/util.py

```diff
@@ -93,12 +93,12 @@ def coerce_to_output(
         # then the result e a Series of tuple/list, or
         # if this is non grouped aggregate, then the result
         return ibis.util.coerce_to_dataframe(result, expr.type().names)
-    elif np.isscalar(result):
+    elif isinstance(result, pd.Series):
+        return result.rename(result_name)
+    else:
```

Comment:
can you make this one an elif

Reply:
Good point. Update this function to be more restrictive.